src/components/Tabs/Tabs.js

```diff
@@ -173,7 +173,7 @@ export default class Tabs extends React.Component {
     const selectedLabel = selectedTab ? selectedTab.props.label : '';
 
     return (
-      <div>
+      <React.Fragment>
```

Comment:
Can we use `Fragment` named import from `react` (as several other components do)?

Reply:
@asudoh thanks! looking at other files to maintain consistency

Reply:
@asudoh looking at other files, it looks like `import` with `<Fragment>` is only used once, but `<React.Fragment` is used 4 or 5 times. sticking with `<React.Fragment>` in this PR

Reply:
@lovemecomputer Thank you for doing the research!